### DISCUSSION TITLE
🧩 FEATURE REQUEST: Upvote the article from `Reading History` page

### DISCUSSION THREAD

**Discussion by omBratteng** [dailydotdev/daily#614] [Feature Requests]
## Feature Request

I'd like to be able to upvote a read article from the `Reading History` page, as I often open many articles and close the feed.
Although it is possible to click into the article from the history page, and then upvote the article.

### Solution

Either a new button with upvote

![image](https://user-images.githubusercontent.com/1681525/161000292-e26476b4-6d4a-4ab6-a778-6ee5304f2f5e.png)

Alternatively, from the dropdown

![image](https://user-images.githubusercontent.com/1681525/161000727-a493e173-3496-4cd1-9b2d-71c8fb359251.png)

**Comment by rebelchris** [comment]
Great idea @omBratteng this would definitely be a great addition.
I'll forward it to the product team.

**Comment by rebelchris** [comment]
Moving this to feature requests for now
